src/course-unit/data/api.js

```diff
@@ -64,17 +64,19 @@ export async function getCourseSectionVerticalData(unitId) {
  * @param {string} options.parentLocator - The parent locator of the XBlock.
  * @param {string} [options.displayName] - The display name for the XBlock.
  * @param {string} [options.boilerplate] - The boilerplate for the XBlock.
+ * @param {string} [options.stagedContent] - The staged content for the XBlock.
```

Comment:
[optional] Repeated parts may be omitted.
```suggestion
 * @param {string} options.parentLocator - The parent locator.
 * @param {string} [options.displayName] - The display name.
 * @param {string} [options.boilerplate] - The boilerplate.
 * @param {string} [options.stagedContent] - The staged content.
```

Reply:
Done